src/components/MainContent.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import { Link } from 'gatsby'
+import { PrismicLink } from '@prismicio/react'
```

Comment:
Can you please remove this file entirely? I forgot to do it before. We do this in this step: https://prismic.io/docs/technologies/tutorial-3-create-homepage-slices-gatsby#2.-update-the-index.js-file